testing/src/main/java/org/oppia/android/testing/network/MockPlatformParameterService.kt

```diff
@@ -40,12 +42,14 @@ class MockPlatformParameterService(
       appVersionForCorrectResponse -> mapOf(
         TEST_STRING_PARAM_NAME to TEST_STRING_PARAM_SERVER_VALUE,
         TEST_INTEGER_PARAM_NAME to TEST_INTEGER_PARAM_SERVER_VALUE,
-        TEST_BOOLEAN_PARAM_NAME to TEST_BOOLEAN_PARAM_SERVER_VALUE
+        TEST_BOOLEAN_PARAM_NAME to TEST_BOOLEAN_PARAM_SERVER_VALUE,
+        SPLASH_SCREEN_WELCOME_MSG to SPLASH_SCREEN_WELCOME_MSG_SERVER_VALUE
       )
       appVersionForWrongResponse -> mapOf(
         TEST_STRING_PARAM_NAME to TEST_STRING_PARAM_SERVER_VALUE,
         TEST_INTEGER_PARAM_NAME to TEST_INTEGER_PARAM_SERVER_VALUE,
-        TEST_BOOLEAN_PARAM_NAME to TEST_OBJECT_AS_PARAM_VALUE // unsupported value type in response
+        TEST_BOOLEAN_PARAM_NAME to TEST_OBJECT_AS_PARAM_VALUE, // unsupported value type in response
```

Comment:
Recommend removing the comment and changing the `TEST_OBJECT_AS_PARAM_VALUE` => `TEST_UNSUPPORTED_OBJECT_AS_PARAM_VALUE`

Reply:
Done